src/app/routes.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+
+import * as routes from './constants/routes';
+
```

Comment:
no need this extra line. However, you could keep one line in between third party imports and our module imports.